### ISSUE TITLE
Testnet panicked

### ISSUE THREAD

**Issue by wkarshat** [ZcashFoundation/parity-zcash#22]
```
ubuntu@vokol:~/zcash/zebra$ ./target/release/zebra --testnet
2019-06-18 12:01:19 UTC INFO sync Processed 1000 blocks in 1.90 seconds (526.19 blk/s). Peers: 3 (act: 3, idl: 0, bad: 0).     Chain: [sch:64 > req:349 > vfy:27 > db:1001]
2019-06-18 12:01:20 UTC INFO sync Processed 1000 blocks in 1.28 seconds (780.35 blk/s). Peers: 3 (act: 3, idl: 0, bad: 0).     Chain: [sch:0 > req:160 > vfy:80 > db:2001]
thread '<unnamed>' panicked at 'slice index starts at 1 but ends at 0', src/libcore/slice/mod.rs:2347:5
stack backtrace:
```

**Comment by wkarshat** [comment]
[zebra.tar.gz](https://github.com/ZcashFoundation/zebra/files/3301402/zebra.tar.gz)

**Comment by dconnolly** [comment]
I think this is related to #12

**Comment by dconnolly** [comment]
@wkarshat I just merged in #35, I'm marking this as fixed unless you don't see it fixed on your end. 🙏
